modules/waypoint_names_to_coordinates.py

```diff
@@ -0,0 +1,59 @@
+def waypoint_names_to_coordinates(waypoint_names: list[str],
```

Comment:
Add docstring at top of file, can be short and simple. Add 2 empty lines between it and this function.